docs/product-assembly_pam-on-openshift/src/main/asciidoc/apb-deprovision-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='apb-deprovision-proc']
+= Deprovisioning a {PRODUCT} environment using the Automation Broker
+
+If you deployed a {PRODUCT} environment using the Automation Broker, you can also use the Broker to deprovision the environment.
+
+.Procedure
+
+. In the OpenShift Web UI, view the Overview page for your project.
+
+. Locate the Service Instance for the {PRODUCT} environment that you want to remove.
+
+. Click the menu button at the instance and select *Delete*. The Automation Broker runs the deprovisioning playbook.
```

Comment:
Is the button labeled "Menu". If so, then "Click **Menu at the instance**". If not, then call it what it's labeled, in bold. If just an icon or a generic button, then fine as "menu button" as the exception I mentioned in my previous comment about saying "button".